timestamp-client/src/main/java/com/palantir/timestamp/RateLimitedTimestampService.java

```diff
@@ -40,7 +40,7 @@
  */
 @ThreadSafe
 public class RateLimitedTimestampService implements TimestampService {
```

Comment:
Did it not work to just reference this class from where it was in `timestamp-impl`? I saw your comment on the PR description but not sure I follow why it had to move